package.json

```diff
@@ -26,14 +26,20 @@
     "jed": "^1.1.1",
     "jshint": "~2.9.1",
     "jshint-loader": "~0.8.3",
+    "less": "^3.7.1",
+    "less-loader": "^4.1.0",
```

Comment:
Such "merge upstream" commits must not appear in PRs, they are essentially impossible to review. Please use `git fetch origin; git rebase -i origin/master` ("rebasing") to put your changes on top of what's already in master, and then adjust *your* commits (not the master ones).